tests/processing/test_processor.py

```diff
@@ -306,13 +311,15 @@ def test_stream_processor_create_with_partitions() -> None:
     assert factory.create_with_partitions.call_count == 1
     assert create_args[1] == offsets_p0
 
+    consumer.tell.return_value = {**offsets_p0}
+
     # Second partition assigned
     offsets_p1 = {Partition(topic, 1): 0}
     assignment_callback(offsets_p1)
 
     create_args, _ = factory.create_with_partitions.call_args
     assert factory.create_with_partitions.call_count == 2
-    assert create_args[1] == offsets_p1
+    assert create_args[1] == {**offsets_p1, **offsets_p0}
```

Comment:
was this test change related to your other changes? since there's no cooperative rebalancing here, seems like the assertions should stay the same?

Reply:
the mocked return value for `consumer.tell` was wrong, so this had the wrong value. the assignments in this test are actually incremental: first p1 is assigned, then p0, and there's no revocation.